tests/ember_debug/profile-manager-test.js

```diff
@@ -1,8 +1,246 @@
-import { test } from 'qunit';
 import ProfileManager from 'ember-debug/models/profile-manager';
+import { find, visit, waitUntil } from '@ember/test-helpers';
+import EmberComponent from '@ember/component';
+import EmberRoute from '@ember/routing/route';
+import EmberObject from '@ember/object';
+import Controller from '@ember/controller';
+import { module, test } from 'qunit';
+import { hbs } from 'ember-cli-htmlbars';
+import EmberDebug from 'ember-debug/main';
+import setupEmberDebugTest from '../helpers/setup-ember-debug-test';
+import { run } from '@ember/runloop';
 
-test('Ember Debug - Construction', function (assert) {
-  let manager = new ProfileManager();
-  assert.ok(!!manager, 'it was created');
-  assert.equal(manager.profiles.length, 0, 'it has no profiles');
+const getRounded = (value) => {
+  let data = value;
+  if (typeof data === 'string') {
+    // remove unit px
+    if (data.indexOf('px') !== -1) {
+      data.replace('px', '');
+    }
+    data = parseFloat(data);
+  }
+  return Math.floor(data);
+};
+
+module('Ember Debug - profile manager component highlight', function (hooks) {
+  setupEmberDebugTest(hooks, {
+    routes() {
+      this.route('simple');
+      this.route('posts', { resetNamespace: true });
+    },
+  });
+
+  hooks.beforeEach(async function () {
+    EmberDebug.IGNORE_DEPRECATIONS = true;
+
+    this.owner.register(
+      'route:application',
+      EmberRoute.extend({
+        model() {
+          return EmberObject.create({
+            toString() {
+              return 'Application model';
+            },
+          });
+        },
+      })
+    );
+
+    this.owner.register(
+      'route:simple',
+      EmberRoute.extend({
+        model() {
+          return EmberObject.create({
+            toString() {
+              return 'Simple Model';
+            },
+          });
+        },
+      })
+    );
+
+    this.owner.register(
+      'route:posts',
+      EmberRoute.extend({
+        model() {
+          return 'String as model';
+        },
+      })
+    );
+
+    this.owner.register(
+      'controller:application',
+      Controller.extend({
+        toString() {
+          return 'App.ApplicationController';
+        },
+      })
+    );
+
+    this.owner.register(
+      'controller:simple',
+      Controller.extend({
+        toString() {
+          return 'App.SimpleController';
+        },
+      })
+    );
+
+    this.owner.register(
+      'component:test-foo',
+      EmberComponent.extend({
+        classNames: ['simple-component'],
+        toString() {
+          return 'App.TestFooComponent';
+        },
+      })
+    );
+
+    this.owner.register(
+      'component:test-bar',
+      EmberComponent.extend({
```

Comment:
do we have ability to register glimmer components here? likely we should check how it will work with multiple roots (1+ higlight nodes should exist)